owncloudApp/src/main/java/com/owncloud/android/presentation/providers/sharing/UsersAndGroupsSearchProvider.kt

```diff
@@ -180,10 +180,7 @@ class UsersAndGroupsSearchProvider : ContentProvider() {
                 suggestAuthority!! + DATA_REMOTE_SUFFIX
             ).build()
 
-            val manager = FileDataStorageManager(context, account, context.contentResolver)
-
-            val federatedShareAllowed = manager.getCapability(account!!.name)?.filesSharingFederationOutgoing
-                ?.isTrue
+            val federatedShareAllowed = capabilities?.filesSharingFederationOutgoing?.isTrue
```

Comment:
If you use it that way, federatedShareAllowed is nullable, maybe we can avoid nullability using it this way:
`val federatedShareAllowed = capabilities?.filesSharingFederationOutgoing?.isTrue ?: false`